default.nix

```diff
@@ -40,15 +44,6 @@ let
     core_kernel
   ];
 
-  # We wrap the upstream tezos-client derivation since it also brings some ocaml
-  # packages which are incompatible with ours.
-  tezosClient =
-    let orig = (import "${sources.tezos-packaging}/nix" { }).binaries.tezos-client;
-    in
-    pkgsLinux.runCommand "tezos-client" { } ''
-      mkdir -p $out/bin; ln -s ${orig}/bin/tezos-client $out/bin/tezos-client
-    '';
```

Comment:
The call to `tezos-client` on `compile-lige.rb` is already surrounded with a try-catch block just for this reason :). So it should be fine if we keep it, it might be useful in future if we start to measure the main contract size again; but I'm happy if we remove it too.

Reply:
Fair point :+1: Thought it might be weird leaving code we know to be inaccessible in, but I am fine either way really :)